05/styles/global.css

```diff
@@ -0,0 +1,54 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+header {
```

Comment:
Warto stosować resete.css, a potem stylować już tylko po klasach. Przy dużych projektach na prawdę to dużo uprasza :)